src/main/java/com/google/api/generator/gapic/composer/SampleCodeHelperComposer.java

```diff
@@ -71,14 +82,45 @@ private static TryCatchStatement composeUnaryRpcMethodSampleCode(
       List<MethodArgument> arguments,
       TypeNode clientType,
       Map<String, ResourceName> resourceNames) {
-    // TODO(summerji): compose sample code for unary rpc method.
     VariableExpr clientVarExpr = createVariableExpr(getClientName(clientType), clientType);
+    // Assign each method arguments with its default value.
+    Map<String, VariableExpr> methodArgVarExprMap = mapMethodArgumentsToVariableExprs(arguments);
+    List<Expr> methodArgumentsAssignmentExpr =
+        assignMethodArgumentsWithDefaultValues(arguments, methodArgVarExprMap, resourceNames);
+    List<Expr> methodVarExprs =
+        arguments.stream()
+            .map(arg -> methodArgVarExprMap.get(arg.name()))
+            .collect(Collectors.toList());
```

Comment:
Regarding to create variable expr once issue, for method arguments, what I did here is to create a Map<String, VariableExpr> for method arguments, where key is argument's name and value is its variable expr.
Then I fetch the values from map to make a `List<Expr>` and reused in the below session. 
What do you think of this approach. Welcome any suggestion. Thanks.